jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -88,13 +88,9 @@ async def _ensure_mount_server(self):
                     # Cannot mount in non-privileged container, so use unshare for a private mount
                     get_logger().info("Non-privileged container...")
                     subprocess.run(['mkdir','-p', f'/mnt{self.mount_dir}'])
-
-                    if self.sock_path:
-                        mount_server_cmd = f"unshare -Ufirm mount-server --mount-dir /mnt{self.mount_dir} --allow-other=false"
-                        mount_server_cmd += f" --sock-path {self.sock_path}"
-                    else:
-                        mount_server_cmd = f"unshare -Ufirm mount-server --mount-dir /mnt{self.mount_dir} --allow-other=false"
-
+                    mount_server_cmd = f"unshare -Ufirm mount-server --mount-dir /mnt{self.mount_dir} --allow-other=false"
+                if self.sock_path:
```

Comment:
Still need to remove lines 85-86

Reply:
I think I will close this PR since Jerry created another PR with additional changes.